cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -88,6 +88,7 @@ spec:
   # enable the crash collector for ceph daemon crash collection
   crashCollector:
     disable: false
+    daysToRetain: 30
```

Comment:
The doc says the default is to retain them indefinitely, but this shows 30. How about commenting this setting out by default so we get the default behavior in the example?